### ISSUE TITLE
AdGuardExtra causes microphone input not to work on voice.google.com

### ISSUE THREAD

**Issue by illgitthat** [AdguardTeam/AdGuardExtra#560]
### Prerequisites

- [X] Filters were updated before reproducing an issue
- [X] AdGuard Extra version is up-to-date
- [X] I checked to make sure that this issue has not already been filed

### What product do you use?

AdGuard Extra Extension

### What type of problem have you encountered?

Website or app doesn't work properly

### Which browser(s) do you use?

Microsoft Edge

### Where is the problem encountered?

https://voice.google.com

### What filters do you have enabled?

AdGuard Base filter

### Issue Details

Steps to reproduce:
1. Go to voice.google.com
2. Start a phone call
3. Notice that no audio input is transmitted

### Expected Behavior

Caller can hear my voice

### Actual Behavior

Caller can't hear me. If I disable the extension and reload, it works fine.

### Screenshots

<details><summary>Screenshot 1</summary>

![image](https://github.com/user-attachments/assets/56ad115d-77be-4358-b248-8d8c00ea85bc)

</details>

### Privacy

- [X] I agree to follow this condition

**Comment by AdamWr** [comment]
Unfortunately, I'm not able to check it, because `voice.google.com` is not available in my country and it doesn't work with VPN.
AdGuard Extra runs only on websites listed here - https://github.com/AdguardTeam/AdGuardExtra/blob/master/dist/hostnames.json so it shouldn't run on `voice.google.com`.
Could you please re-check and ensure that it's caused by AdGuard Extra and not by another extension?

**Comment by illgitthat** [comment]
> Unfortunately, I'm not able to check it, because `voice.google.com` is not available in my country and it doesn't work with VPN. AdGuard Extra runs only on websites listed here - https://github.com/AdguardTeam/AdGuardExtra/blob/master/dist/hostnames.json so it shouldn't run on `voice.google.com`. Could you please re-check and ensure that it's caused by AdGuard Extra and not by another extension?

I agree. I had checked the same list of hostnames and saw it wasn't on there. However I am sure that it was the extension because disabling it immediately fixed the issue. In any case, I can't reproduce it now so I must be crazy. Closing this out and will reopen if I come across anything similar again.
